blocklyc.js

```diff
@@ -475,6 +476,7 @@ function cloudCompile(text, action, successHandler) {
                 } else {
                     var loadWaitMsg = (action !== 'compile') ? '\nDownload...' : '';
                     $("#compile-console").val($("#compile-console").val() + data['message'] + loadWaitMsg);
+    console.log(data);                
```

Comment:
This appears to be debugging code. Can this be safely removed?

Reply:
Same, this can be removed.